test/accessibility.html

```diff
@@ -221,21 +221,30 @@
             });
 
             it('should not be marked required', function() {
-              expect(input.hasAttribute('aria-required')).to.be.false;
+              expect(input.hasAttribute('required')).to.be.false;
             });
 
             it('should be marked required', function() {
               tf.required = true;
-              expect(input.hasAttribute('aria-required')).to.be.true;
+              expect(input.getAttribute('required')).to.equal('');
```

Comment:
Explicitly checking for empty string value instead of using `hasAttribute` so that it would fail if the attribute would have some unexpected value instead.